server/api/student/student.model.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
+
+var schoolYearSchema = new Schema({
+	allAbsences: Number,
+	tardy: Number,
+	present: Number,
+	enrolled: Number
+});
+
+var StudentSchema = new Schema({
+  _id: Number, // Student ID
+  lastName: String,
+  firstName: String,
+  schoolYear: [ Number: {schoolYearSchema}]
```

Comment:
- This has worked for me in the past. So I figure we could try it in this project. 

- Year would go under 'year' in SchoolYearSchema